File: tests/fixtures/csharp/ProductionApiService.cs
class AppConfig {
    public string Host { get; set; }
    public int Port { get; set; }
    public string DatabaseUrl { get; set; }
    public string RedisUrl { get; set; }
    public string JwtSecret { get; set; }
    public string LogLevel { get; set; }
    public int MaxConnections { get; set; }
    public int TimeoutSecs { get; set; }
    public int RetryCount { get; set; }
    public int CacheTtl { get; set; }
    public int RateLimit { get; set; }
    public string UploadDir { get; set; }
    public string CorsOrigin { get; set; }
}

class RequestHandler {
    string Dispatch(string action) {
        switch (action) {
        case "create":
            return "creating";
        case "delete":
            return "deleting";
        case "update":
            return "updating";
        case "reset":
            return "resetting";
        case "notify":
            return "notifying";
        case "export":
            return "exporting";
        default:
            return "unknown";
        }
    }

    int ProcessEvent(int[] data) {
        int a = data[0];
        int b = data[1];
        int c = data[2];
        int d = data[3];
        int e = data[4];
        int f = data[5];
        int g = data[6];
        int h = data[7];
        if (a > 100) {
            return -1;
        }
        if (b > 200) {
            return -2;
        }
        int result = a + b + c + d;
        int extra = e + f + g + h;
        return result + extra;
    }
}

File: tests/fixtures/kotlin/ProductionApiService.kt
data class AppConfig(
    val host: String,
    val port: Int,
    val databaseUrl: String,
    val redisUrl: String,
    val jwtSecret: String,
    val logLevel: String,
    val maxConnections: Int,
    val timeoutSecs: Int,
    val retryCount: Int,
    val cacheTtl: Int,
    val rateLimit: Int,
    val uploadDir: String,
    val corsOrigin: String
)

class RequestHandler {
    fun dispatch(action: String): String {
        return when (action) {
            "create" -> "creating"
            "delete" -> "deleting"
            "update" -> "updating"
            "reset" -> "resetting"
            "notify" -> "notifying"
            "export" -> "exporting"
            else -> "unknown"
        }
    }

    fun processEvent(data: IntArray): Int {
        val a = data[0]
        val b = data[1]
        val c = data[2]
        val d = data[3]
        val e = data[4]
        val f = data[5]
        val g = data[6]
        val h = data[7]
        if (a > 100) {
            return -1
        }
        if (b > 200) {
            return -2
        }
        val result = a + b + c + d
        val extra = e + f + g + h
        return result + extra
    }
}
